test/kubemark/start-kubemark.sh

```diff
@@ -368,12 +94,12 @@ current-context: kubemark-context"
 
   # Create secret for passing kubeconfigs to kubelet, kubeproxy and npd.
   "${KUBECTL}" create secret generic "kubeconfig" --type=Opaque --namespace="kubemark" \
-    --from-literal=kubelet.kubeconfig="${KUBELET_KUBECONFIG_CONTENTS}" \
-    --from-literal=kubeproxy.kubeconfig="${KUBEPROXY_KUBECONFIG_CONTENTS}" \
-    --from-literal=heapster.kubeconfig="${HEAPSTER_KUBECONFIG_CONTENTS}" \
-    --from-literal=cluster_autoscaler.kubeconfig="${CLUSTER_AUTOSCALER_KUBECONFIG_CONTENTS}" \
-    --from-literal=npd.kubeconfig="${NPD_KUBECONFIG_CONTENTS}" \
-    --from-literal=dns.kubeconfig="${KUBE_DNS_KUBECONFIG_CONTENTS}"
+    --from-file=kubelet.kubeconfig="${LOCAL_KUBECONFIG}" \
```

Comment:
Previously we had different (user, token) set for every component.
I think it has a value by itself, and we should try to mimic such behavior (given it's how it looks in real cluster).

I don't want to block this PR on it (because it's great on its own), but please file an issue and add a TODO to fix that.

Reply:
Added comment.